src/components/DataTable/DataTableDataCell.tsx

```diff
@@ -19,10 +19,39 @@ const DataTableDataCell = ({
 }: DataCellProps) => {
   const anchorEl = React.useRef(null);
   const [showMenu, setShowMenu] = useState(false);
+  const [textContent, setTextContent] = useState("");
+
+  useEffect(() => {
```

Comment:
Suggestion; it might be worth adding some tests for this?

Reply:
i've added some tests:)